corehq/apps/analytics/tasks.py

```diff
@@ -140,7 +141,7 @@ def _get_client_ip(meta):
     return ip
 
 
-def _send_form_to_hubspot(form_id, webuser, cookies, meta, extra_fields=None):
+def _send_form_to_hubspot(form_id, webuser, cookies, meta, extra_fields=None, prelogin=False):
```

Comment:
I see your point about not wanting `extra_fields` in both calls, and having the most common use case require fewer parameters. I still don't like having `webuser`'s data type vary. How about changing `prelogin` from a bool to a string (`email`?), and passing `None` to `webuser` when it's provided?